### PR TITLE
Bftools update

### PR DESCRIPTION
**Pull request by bramalingam**
ref: https://github.com/openmicroscopy/bioformats/pull/1778

Adding the remaining parts to complete the fixes.

Testing Instructions:
1) Build bioformats locally.
2) Now remove : loci_tools.jar and bioformats_plugins.jar from the artifacts folder.
3) Run ./ijview  "path/ to/ image" , 
ex: tools bramalingam$ ./ijview /Users/bramalingam/Desktop/IMG_1355.JPG 
With this PR in,the image should open in IJ without any issues.

Without this PR you should get the following exception with the above workflow,
``
Error: Could not find or load main class loci.plugins.in.Importer
``

### CODE REVIEW

File: tools/bf.bat [hunk 1]
```diff
@@ -48,12 +48,12 @@ if not "%BF_DEVEL%" == "" (
...
 rem Developer environment variable unset; add JAR libraries to classpath.
-if exist "%BF_JAR_DIR%\formats-gpl.jar" (
-  set BF_CP=%BF_CP%;"%BF_JAR_DIR%\formats-gpl.jar";"%BF_JAR_DIR%\bio-formats-tools.jar"
-) else if exist "%BF_JAR_DIR%\bioformats_package.jar" (
-  set BF_CP=%BF_CP%;"%BF_JAR_DIR%\bioformats_package.jar"
-) else if exist "%BF_JAR_DIR%\loci_tools.jar" (
-  set BF_CP=%BF_CP%;"%BF_JAR_DIR%\loci_tools.jar"
+if exist "%BF_JAR_DIR%\bioformats_package.jar" (
+    set BF_CP=%BF_CP%;"%BF_JAR_DIR%\bioformats_package.jar"  
+) else  if exist "%BF_JAR_DIR%\loci_tools.jar"(
```
Comment (mtbc) [inline tools/bf.bat]: some trailing space on these lines

Comment (mtbc) [inline tools/bf.bat]: also double space after "else"

**Comment by mtbc** [comment]
Works just as described. :+1: I couldn't see a `bioformats_plugins.jar` so I instead deleted `bioformats_package.jar` for that one.

**Comment by bramalingam** [comment]
@melissalinkert : Can you check the last commit? I'm not sure if I've implemented it right.

**Comment by melissalinkert** [comment]
I'd rather not have the garbage collection changed automatically.  I think the more flexible thing to do is what's proposed at the end of the ome-users thread:

```
if [ -z "$BF_FLAGS" ]
then
  BF_FLAGS=""
fi
```
and then update the ```BF_FLAGS="-Xmx$BF_MAX_MEM"``` line to be ```BF_FLAGS="$BF_FLAGS -Xmx$BF_MAX_MEM"``` (in bf.sh; similar changes would be needed in bf.bat).  That lets the user pass any additional options to the command line tools by setting ```BF_FLAGS``` externally.

**Comment by rleigh-codelibre** [comment]
With bftools.jar from the current merge build, on Windows 8.1 x64 with Java 8u45 x64:

```
(27) C:\Users\rleigh\Desktop\bftools-cur>showinf
The syntax of the command is incorrect.

(27) C:\Users\rleigh\Desktop\bftools-cur>bf
The syntax of the command is incorrect.
```

**Comment by bramalingam** [comment]
@melissalinkert : can you please take another look?

**Comment by melissalinkert** [comment]
Line 22 in bf.sh and 23 in bf.bat shouldn't be necessary, but otherwise that looks much better.

**Comment by mtbc** [comment]
Works fine functionally on Linux. :+1:

**Comment by rleigh-codelibre** [comment]
Still broken on Windows:

```
(27) C:\Users\rleigh\Desktop\bftools>showinf
Error: Could not find or load main class
```

**Comment by mtbc** [comment]
After much Windows pain this morning (I keep thinking, I should make myself use it more, then when I do it reminds me why I don't): confirmed, I get exactly the same message as @rleigh-dundee.

**Comment by melissalinkert** [comment]
PR opened against this to fix Windows: https://github.com/bramalingam/bioformats/pull/5

**Comment by melissalinkert** [comment]
@rleigh-codelibre/@rleigh-dundee/@mtbc - could one of you please confirm that this now works for you on Windows?

**Comment by rleigh-dundee** [comment]
@melissalinkert showinf and bfconvert are working on Windows for me with today's merge build.

**Pull request merged**